projects/plugins/jetpack/modules/widgets/search.php

```diff
@@ -1,4 +1,4 @@
-<?php
+<?php // phpcs:ignore WordPress.Files.FileName.InvalidClassFileName
```

Comment:
Could we bring class `Jetpack_Search_Widget` over btw? only leave the init process in this file.

Reply:
The widget will likely migrate during milestone 2 :)